SR2020/2020-05-26-virtual-competition-dropped-out-teams.md

```diff
@@ -0,0 +1,40 @@
+---
+to: sr2020-teams
+subject: Would you like to compete in the Student Robotics 2020 Virtual Competition
+---
+
+![](https://studentrobotics.org/docs/resources/competition-simulator/arena.png)
```

Comment:
Not sure if this is just a limitation of markdown, but we should ensure that this doesn't fill the screen for recipients (as it does in the preview on GitHub). It may also make sense to embed it as a (downsized) inline image rather than a link to a huge external one.

Reply:
![image](https://user-images.githubusercontent.com/6527489/84569311-74c7f480-ad7d-11ea-83b7-575dc9fe934a.png)

Here's how it looks in Gmail. I think this is fine.